src/placeos-driver/interface/sensor.cr

```diff
@@ -141,6 +141,9 @@ abstract class PlaceOS::Driver
       property module_id : String?
       property binding : String?
 
+      # The unit type string as supported by crunits
+      property unit : String?
```

Comment:
Is there an enum for this rather?

Reply:
no as you can dynamically define new units - and the units are complex compound expressions
like `m/s^2`